front/src/styles/scss/applications/operationalStudies/_managetrainschedule.scss

```diff
@@ -42,6 +42,16 @@
   }
 }
 
+.flex-on-line {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
```

Comment:
Generic scss should be in styles/scss/common

You could just use classes `d-flex align-items-center` and just add `gap: 10px,` to the file

Reply:
moved, creeeated a gap-10 class